### PR TITLE
Add Edward Skeletal PPP

### PR DESCRIPTION
**Pull request by ThePrevailingOne**

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -31,14 +31,16 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 I am an eager Y2 computer science undergraduate with the desire to learn and improve myself in this field. I aim to delve deeper into the world of software engineering and cybersecurity which address and solve real world problems with the focus on privacy and data security.
 
-### Johnny Doe
+### Edward Alvin
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/Edward_Alvin.jpeg" width="200px">
 
-[[github](http://github.com/johndoe)] [[portfolio](team/johndoe.md)]
```
Comment (Graphcalibur) [inline docs/AboutUs.md:38]: The portfolio link I believe should be kept as it is supposed to link to the PPP

**Review by Graphcalibur — CHANGES_REQUESTED** [review]

**Comment by ThePrevailingOne** [comment]
Can check if the update is already right

File: docs/team/theprevailingone.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+applied to for internships as well as any events you've arranged with them. It is optimized for 
+CLI users who prefer to type commands for efficiency.
+
+Given below are my contributions to the project.
+
+* **New Feature**: Added the ability to undo/redo previous commands.
+  * What it does: 
+  * Justification: 
+  * Highlights: 
+  * Credits: *{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}*
```
Comment (albertsutz) [inline docs/team/theprevailingone.md:18]: Can remove this first. Every other things looks good!

**Review by albertsutz — CHANGES_REQUESTED** [review]

**Review by albertsutz — APPROVED** [review]

**Review by Graphcalibur — APPROVED** [review]
LGTM!

**Pull request merged**